msteams-platform/resources/design/framework/personal-apps.md

```diff
@@ -2,10 +2,12 @@
 title: Design Guidelines Reference
 description: Describes the guidelines for designing a personal app
 keywords: teams design guidelines reference framework personal apps
-ms.date: 04/30/2018
 --- 
 # Personal apps
 
+> [!Important]
+> Full support for personal apps on mobile clients is currently in [developer preview](~/resources/dev-preview/developer-preview-intro.md), and will be released soon. To prepare for this change you should follow the [guidance for personal apps on mobile](~/resources/design/framework/tabs-mobile.md) when creating your tabs. When full support for tabs is release, your `contentUrl` **will be loaded in the mobile Teams client**. Your users will still have the option of opening your tab in a separate browser via your `websiteUrl`, however your `contentUrl` will be loaded first.
```

Comment:
Please remove below statement as static tabs won't have an option to open in browser:
Your users will still have the option of opening your tab in a separate browser via your `websiteUrl`, however your `contentUrl` will be loaded first.